packages/block-editor/src/components/media-flow/style.scss

```diff
@@ -1,5 +1,10 @@
+.components-popover:not(.is-mobile).is-bottom {
+	z-index: 1000000 !important;
```

Comment:
why is this necessary & why can't a normal value from the z-index mixin be here?